forecast/preprocessor.py

```diff
@@ -53,7 +53,7 @@ class Preprocessor:
         "query_pos",
         "location",
         "application_name",
-        "backend_type",
+        #"backend_type",
```

Comment:
Out of curiosity, why is this commented and others aren't?

If it is to save on memory or to address PostgreSQL query log compatibility for tiramisu, that's OK -- we should just target PostgreSQL 14. Add flags to programmatically disable some columns for older versions if you want.

Reply:
Where should these flags be? in the preprocessor itself?

Reply:
Thoughts on:

Preprocessor now accepts a list of the columns in the constructor;
PreprocessorCLI is what sets the list of columns based on what flags are passed in

?

Reply:
That makes sense. Will do this